Error during compilation

I'm getting the following error with ctru-rs via rust3ds-template:
error[E0200]: the trait `core::ops::Place<V>` requires an `unsafe impl` declaration
    --> /home/XX/.cargo/git/checkouts/ctru-rs-1206856f7a5d76bc/bf6ef66/ctr-std/src/collections/hash/map.rs:1975:1
     |
1975 | / impl<'a, K, V> Place<V> for EntryPlace<'a, K, V> {
1976 | |     fn pointer(&mut self) -> *mut V {
1977 | |         self.bucket.read_mut().1
1978 | |     }
1979 | | }
     | |_^

error: aborting due to previous error

error: Could not compile `std`.


My version:
X@anonymous:~/3DS/rust3ds-template$ rustc -V
rustc 1.25.0-nightly (9fd7da904 2018-01-25)
X@anonymous:~/3DS/rust3ds-template$ xargo -V
xargo 0.3.10
cargo 0.26.0-nightly (91e36aa86 2018-01-25)

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,7 @@
 language: rust
 
 rust:
-   - nightly-2018-01-21
+   - nightly-2018-01-26
    - nightly
 
 matrix:

--- a/ctr-std/src/collections/hash/map.rs
+++ b/ctr-std/src/collections/hash/map.rs
@@ -1972,7 +1972,7 @@ impl<'a, K, V> Placer<V> for Entry<'a, K, V> {
 #[unstable(feature = "collection_placement",
            reason = "placement protocol is subject to change",
            issue = "30172")]
-impl<'a, K, V> Place<V> for EntryPlace<'a, K, V> {
+unsafe impl<'a, K, V> Place<V> for EntryPlace<'a, K, V> {
     fn pointer(&mut self) -> *mut V {
         self.bucket.read_mut().1
     }

--- a/ctr-std/src/lib.rs
+++ b/ctr-std/src/lib.rs
@@ -296,7 +296,6 @@
 #![feature(ptr_internals)]
 #![feature(rand)]
 #![feature(raw)]
-#![feature(repr_align)]
 #![feature(rustc_attrs)]
 #![feature(sip_hash_13)]
 #![feature(slice_bytes)]

